ministryofjustice/analytics-platform-control-panel
# Fix basic auth

## What

* JWT authentication no longer raises an exception if the `Authorization` header value does not start with `JWT` - this was breaking out of the authentication process instead of passing on to the next method

## How to review

1. Run the api (make sure you have a superuser)
2. Make an HTTP request to http://localhost:8000/users with an `Authorization` header value, eg:
```python
import requests
r = requests.get('http://localhost:8000/users', auth=(username, password))
assert r.status_code == 200
```
3. See that you get a 200 response and not a 403
